kbase-extension/static/kbase/gallery/common.css

```diff
@@ -0,0 +1,20 @@
+.Demo {
```

Comment:
if we're going to have a gallery, may as well make it official and integrate this into the common CSS file.

Reply:
Okay, good idea; css has been moved to `_gallery.scss` in `partials`. Was trying to keep the gallery decoupled (it depends on the Narrative source, but Narrative source doesn't know know about it.)